src/video/movies.cpp

```diff
@@ -57,7 +57,13 @@ uint32_t movie_frames;
 uint32_t movie_width, movie_height;
 double movie_fps;
 double movie_duration;
-bool movie_jpeg_range = false;
+bool fullrange_input = false;
+bool okpixelformat = false;
+bool okcolorspace = false;
+bool yuvjfixneeded = false;
+bool use170Mgamma = false;
+bool isdeepandtv = false;
+AVPixelFormat targetpixelformat = AV_PIX_FMT_YUV444P;
```

Comment:
Do we need all these flags as global vars? Can we put some of them locally to the function where they are used?

Reply:
Indeed, they do not. I relocated those that could be.

Reply:
Thanks